<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCropper</class>
 <widget class="QMainWindow" name="ImageCropper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Cropper</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="crop_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resolutions</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>131</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="option4K">
       <property name="text">
        <string>4096x4096</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="option2K">
       <property name="text">
        <string>2048x2048</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="option1K">
       <property name="text">
        <string>1024x1024</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="option4K_2K">
       <property name="text">
        <string>4096x2048</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="option2K_4K">
       <property name="text">
        <string>2048x4096</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="option2K_1K">
       <property name="text">
        <string>2048x1024</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="option1K_2K">
       <property name="text">
        <string>1024x2048</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>391</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Crop resolution</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="crop_width">
         <property name="maxLength">
          <number>4</number>
         </property>
         <property name="cursorPosition">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="crop_height">
         <property name="maxLength">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Images to crop</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="DragDropListView" name="listWidget">
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="remove_item">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove item</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DragDropListView</class>
   <extends>QListWidget</extends>
   <header>dragdroplistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
